pkg/kube/kube_fake.go

```diff
@@ -50,7 +50,7 @@ import (
 	autoscalingv1 "k8s.io/api/autoscaling/v1"
 	batchv1 "k8s.io/api/batch/v1"
 	batchv1b1 "k8s.io/api/batch/v1beta1"
-	csr "k8s.io/api/certificates/v1beta1"
```

Comment:
Can we add cert v1 in addition to v1beta1 and support both?

Reply:
For the test harness?

Reply:
Maybe tests can be on v1, but the Isopod API should be able to support both for versions 1.19-1.22